### PR TITLE
Add development bundle for bionic-rocky

### PR DESCRIPTION
**Pull request by fnordahl**

### CODE REVIEW

File: development/openstack-base-bionic-rocky/README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+ - A minimum of 8GB of physical RAM.
+ - Enough CPU cores to support your capacity requirements.
+ - Two disks (identified by /dev/sda and /dev/sdb); the first is used by MAAS for the OS install, the second for Ceph storage.
+ - Two cabled network ports on eno1 and eno2 (see below).
+
+Servers should have two physical network ports cabled; the first is used for general communication between services in the Cloud, the second is used for 'public' network traffic to and from instances (North/South traffic) running within the Cloud.
+
+## Components
+
+ - 1 Node for Neutron Gateway and Ceph with RabbitMQ and MySQL under LXC containers.
```
Comment (ChrisMacNaughton) [inline development/openstack-base-bionic-rocky/README.md]: This says that ceph should be on the neutron gateway node as well, while the bundle, below, does not place ceph on that node

Comment (fnordahl) [reply development/openstack-base-bionic-rocky/README.md]: Strictly speaking it does place Ceph, to be specific the Ceph RADOS Gateway, on the Neutron gateway node. :-P

**Comment by sfeole** [comment]
+1 to approve as well. As I've used @fnordahl's bundle guide for rockey/bionic.

**Comment by andrewdmcleod** [comment]
@fnordahl validate-tools is failing so probably needs a tool resync

**Comment by fnordahl** [comment]
I have a PR up in #96 that syncs tools, if we can get that landed I can re-base this one @andrewdmcleod @sfeole

**Comment by sfeole** [comment]
@fnordahl Done, should be good to rebase now

**Pull request merged**